components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/constants/SQLConstants.java

```diff
@@ -419,6 +419,21 @@
                     "UPDATED_TIME) " +
             " VALUES (?,?,?,?,?,?,?)";
 
+    public static final String ADD_MONETIZATION_USAGE_PUBLISH_INFO =
+            " INSERT" +
+                    " INTO AM_MONETIZATION_USAGE_PUBLISHER (ID, STATE, STATUS, STARTED_TIME, LAST_PUBLISHED_TIME) " +
+                    " VALUES (?,?,?,?,?)";
+
+    public static final String UPDATE_MONETIZATION_USAGE_PUBLISH_INFO =
+            " UPDATE AM_MONETIZATION_USAGE_PUBLISHER SET" +
+                    " STATE = ?, STATUS = ?, STARTED_TIME = ?, LAST_PUBLISHED_TIME = ?" +
+                    " WHERE ID = ?";
+
+    public static final String GET_MONETIZATION_USAGE_PUBLISH_INFO =
+            " SELECT " +
+                    "ID, STATE, STATUS, STARTED_TIME, LAST_PUBLISHED_TIME " +
```

Comment:
can move up the second line

Reply:
Changed in https://github.com/wso2/carbon-apimgt/pull/7953/commits/0534e60750b72327d601d99cbdda5c9dbaad001d